CMakeLists.txt

```diff
@@ -55,41 +55,13 @@ endif(MSVC)
 target_link_libraries(ofx ${OpenSP_LIBRARIES} ${ICONV_LIBRARIES})
 
 # ofxconnect
-find_package(CURL)
```

Comment:
Best to leave all of the `find_package` calls in the root CMakeLists.txt, ideally grouped together, to make it easy for builders to determine all of the package's dependencies.

Reply:
I was debating between doing that and indeed was going to bring all the `find_package` calls back to main file, as there aren't too many.